packages/remix-forms/src/createForm.tsx

```diff
@@ -383,7 +396,7 @@ function createForm({
         {Object.keys(schemaShape)
           .map(makeField)
           .map((field) => renderField({ Field, ...field }))}
-        {globalErrors?.length && (
+        {globalErrors?.length > 0 && (
```

Comment:
Now globalErrors will not be `undefined` anymore, but I didn't see any other issues with it being an empty array

Reply:
I will take an empty list and a `.length > 0` over some undefined and falsy evaluation any day of the week :P

Reply:
Actually we need to take control over the amount of undefineds in this code base, or else it will control us.

Reply:
@danielweinmann has a point in that changing from `undefined` to an empty array might be a breaking change since users won't be able to simply test for falsy `errors`